pkg/bbgo/session.go

```diff
@@ -35,6 +35,7 @@ type StandardIndicatorSet struct {
 	ewma  map[types.IntervalWindow]*indicator.EWMA
 	boll  map[types.IntervalWindow]*indicator.BOLL
 	stoch map[types.IntervalWindow]*indicator.STOCH
+	vol   map[types.IntervalWindow]*indicator.VOL
```

Comment:
call it volatility ? because vol can also be volume

Reply:
Sure done!